auto_tests/query_test.c

```diff
@@ -0,0 +1,396 @@
+#ifdef HAVE_CONFIG_H
+#include "config.h"
+#endif
+
+#include <check.h>
+
+#include "../toxcore/DHT.h"
+// #include "../toxcore/DHT.c"
+#include "../toxcore/tox.h"
+#include "../toxcore/Messenger.h"
+#include "../toxcore/query.c"
+
+#include "helpers.h"
+
+#if defined(_WIN32) || defined(__WIN32__) || defined (WIN32)
+#include <windows.h>
+#define c_sleep(x) Sleep(1*x)
+#else
+#include <unistd.h>
+#define c_sleep(x) usleep(1000*x)
+#endif
+
+static unsigned int response_cookie = 348394;
+static uint8_t name[] = "requested_user.this_domain.tld";
+static size_t name_length = strlen("requested_user.this_domain.tld") - 1;
+static uint8_t toxid[TOX_ADDRESS_SIZE] = {0xDD, 0x2A, 0x13, 0x40, 0xFE, 0xCE, 0x44, 0x12, 0x06, 0xB5, 0xF3,
```

Comment:
This function is more or less equivalent to `return false`. As a side-effect it also sets the error code, but it does nothing useful. What is its purpose?